source/docs/software/advanced-controls/geometry/pose.rst

```diff
@@ -13,7 +13,7 @@ You can get the distance to another ``Translation2d`` object by using the ``getD
 Rotation
 --------
 
-Rotation in 2 dimensions is representated by WPILib's ``Rotation2d`` class (`Java <https://github.wpilib.org/allwpilib/docs/beta/java/edu/wpi/first/math/geometry/Rotation2d.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/beta/cpp/classfrc_1_1_rotation2d.html>`__). This class has an angle component, which represents the robot's rotation relative to an axis on a 2-dimensional coordinate system. Due to the angle being stored as its sine and cosine components, the angle will always be in the :math:`\left(-\pi, \pi\right]` range. Positive rotations are counterclockwise.
```

Comment:
Rather then a straight revert, it's probably better to describe here that there are no limits

Reply:
Good point, added in a note.